ruby/santa.rb

```diff
@@ -0,0 +1,97 @@
+# Declared a Santa class
+class Santa
+
+  attr_reader :age, :ethnicity
+  attr_accessor :gender
```

Comment:
Hi @garimaj2108 

Our setter method exists with the `attr_accessor` here.  To get just the setter method you can choose to use the `attr_writer`  generally we want both the setter and getter when using a setter which is what make the `accessor` a popular choice.

The setter method is a shortcut for the instance method of the attribute being added to your attr_ helper method.  In this case

```
def gender=(gender)
  gender = gender
end
```

above is what your setter method is replacing so that we do not need to add it to the code.

Mind the weird syntax of the method - `gender=` method name - `(gender)` parameter

Try this in irb

```
x=(1)
```
then try
```
x = 1
```
let me know what you think and why both work.